src/main/java/seedu/address/logic/commands/event/ViewUpcomingEventsCommand.java

```diff
@@ -16,11 +16,11 @@
 public class ViewUpcomingEventsCommand extends EventCommand {
     public static final String COMMAND_OPTION = "u";
 
-    public static final String MESSAGE_HELP = "Lists upcoming events in NUScheduler for the next specified "
+    public static final String MESSAGE_HELP = "Lists upcoming events in NUScheduler starting in the next specified "
```

Comment:
I feel like this phrasing may not be clear enough. perhaps you could elaborate like so "i.e. events starting today will not be included"